core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFormatterUtil.java

```diff
@@ -62,6 +62,7 @@ interface DateTimeFormatHandler {
       .put("%W", (date) -> "EEEE") // %W => EEEE - Weekday name (Sunday..Saturday)
       .put("%Y", (date) -> "yyyy") // %Y => yyyy - Year, numeric, 4 digits
       .put("%y", (date) -> "yy") // %y => yy - Year, numeric, 2 digits
+      .put("%Q", (date) -> "Q") // %Q => Q - Quarter, numeric, 1 digit
```

Comment:
This is not needed any more I guess

Reply:
Removed in https://github.com/opensearch-project/sql/pull/1421/commits/a38212cbef93f40de3e26ff6628fcfdd9b9660c0